apps/benchmark/gpu_imagenet_bench.py

```diff
@@ -1,80 +1,60 @@
-""" Benchmark script for performance on GPUs.
-
-For example, run the file with:
-`python gpu_imagenet_bench.py --model=mobilenet --target=cuda`.
-For more details about how to set up the inference environment on GPUs,
-please refer to NNVM Tutorial: ImageNet Inference on the GPU
+"""Benchmark script for NVIDIA GPU.
+see README.md for the usage and results of this script.
```

Comment:
Looks like README.md has not been updated for this script? It only has ARM usage.